src/calculateRentalCost.js

```diff
@@ -4,7 +4,23 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const longTermDiscount = 50;
+  const shortTermDiscount = 20;
+  const defaultRentalPrice = 40;
+  const longTerm = 7;
+  const shortTerm = 3;
+
+  const basePrice = days * defaultRentalPrice;
+
+  if (days >= longTerm) {
+    return basePrice - longTermDiscount;
```

Comment:
The discount for rentals of 7 or more days should be applied per day, not as a flat rate. Consider multiplying the `longTermDiscount` by the number of days to apply the discount correctly.